### ISSUE TITLE
Tooltips and Mouse events not triggering on pie chart

### ISSUE THREAD

**Issue by alanhamlett** [c3js/c3#2354]
Tooltips and Mouse [hover and click events](http://c3js.org/reference.html#data-onclick) aren't working on pie charts.

For example, the official pie chart example from docs doesn't show any console logs from mouse events:
http://c3js.org/samples/chart_pie.html

* **C3 version**: 0.5.3
* **D3 version**: 4.13.0
* **Browser**: Chrome 66.0.3359.81
* **OS**: Mac OS X

**Comment by alanhamlett** [comment]
Adding this CSS/Sass fixed the mouse events and tooltips:

```sass
g.c3-chart {
  g.c3-event-rects {
    rect.c3-event-rect {
      pointer-events: none;
    }
  }
}
```

**Comment by prathik457** [comment]
Can we merge this PR, if this works?

**Comment by alpeshv** [comment]
Adding this css breaks tooltip or bar chart.

**Comment by alanhamlett** [comment]
> Adding this css breaks tooltips on bar chart.

Yea, a better solution is needed to work for all c3 charts.

**Comment by alpeshv** [comment]
As @kvcrawford  has suggested, the issue is `.c3-event-rects` getting rendered as last element. His PR should fix this issue for all the charts. PR got approved. Not sure when its going to be merged.

**Comment by 1Jesper1** [comment]
I would like to see this fixed.

**Comment by kt3k** [comment]
Hello @1Jesper1 
We've just published the fix as [v0.6.4](https://github.com/c3js/c3/releases/tag/v0.6.4)! Thanks!
